week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,6 +1,7 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+  // Write the body of this function to concatenate three words together
+  return firstWord.concat(secondWord).concat(thirdWord);
```

Comment:
You can get rid of the last `.concat()` method and have the same output. Can you think of a way how? :)